ykcapi/yk_testing.h

```diff
@@ -3,7 +3,40 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <yk.h>
 
+/// The statistics passed to the `test` function in `ykstats_wait_until`.
+// Note: this struct *must* stay in sync with `YkCStats` in `ykstats.rs`.
+typedef struct {
+  /// How many traces were recorded successfully?
+  uint64_t traces_recorded_ok;
+  /// How many traces were recorded unsuccessfully?
+  uint64_t traces_recorded_err;
+  /// How many traces were compiled successfully?
+  uint64_t traces_compiled_ok;
+  /// How many traces were compiled unsuccessfully?
+  uint64_t traces_compiled_err;
+  /// How many times have traces been executed? Note that the same trace can
+  /// count arbitrarily many times to this.
```

Comment:
I'm not sure what is meant by:

 > Note that the same trace can count arbitrarily many times to this.

Reply:
1 trace can be executed n times where n>=0.

Reply:
Yes.